### PR TITLE
More translations, extracted items into locale-files, added translation_keys to items

### PR DESCRIPTION
**Pull request by Vortilion**

### CODE REVIEW

File: .history/src/locales/de_DE_20240318172510.json
Comment (marcojanssen) [inline .history/src/locales/de_DE_20240318172510.json:1]: Could you remove these from the pull request?

File: src/locales/de_DE.json [hunk 1]
```diff
@@ -7,14 +7,402 @@
...
+    "overlord": "Leutnant",
     "error": {
-      "no-other-monster-available": "Kein anderer Monster verfügbar.",
+      "no-other-monster-available": "Kein anderes Monster verfügbar.",
       "no-other-commander-available": "Kein anderer Kommandant / Leutnant verfügbar."
     }
   },
+  "weapon": {
+    "core": {
+      "amiran-crossbow": "",
```
Comment (marcojanssen) [inline src/locales/de_DE.json]: By leaving this empty it's probably also empty as translation in the app itself. Have you checked that?

File: src/data/repository/ItemData.ts [hunk 2]
```diff
@@ -10,6 +10,7 @@ export interface ItemData {
   name: string;
   itemType: ItemType;
   cardType: CardType;
+  translation_key: string;
```
Comment (marcojanssen) [inline src/data/repository/ItemData.ts:12]: Shouldn't the name be removed from the interface considering you removed that from every item?

**Comment by marcojanssen** [comment]
Great work, some minor comments. I will check how it stores the data (should be by id)

**Comment by marcojanssen** [comment]
Tests failed. You can run these locally with
 npm run test:e2e:dev

**Review by marcojanssen — CHANGES_REQUESTED** [review]

File: src/locales/de_DE.json (hunk 1 above)
Comment (Vortilion) [reply src/locales/de_DE.json]: Yeah, will copy in the english texts for now

File: src/data/repository/ItemData.ts (hunk 2 above)
Comment (Vortilion) [reply src/data/repository/ItemData.ts:12]: yes, absolutely.

Comment (Vortilion) [reply src/data/repository/ItemData.ts:13]: Yes

File: .history/src/locales/de_DE_20240318172510.json
Comment (Vortilion) [reply .history/src/locales/de_DE_20240318172510.json:1]: done.

**Review by marcojanssen — APPROVED** [review]

**Pull request merged**